solutions/cf-1398C.mdx

```diff
@@ -21,33 +21,85 @@ using pi = pair<ll, ll>;
 void solve(){
 	int n; cin >> n;
 	vector<int> dp(n+1, 0);
-	for(int i=1; i<=n; i++){
+	for(int i=1; i<=n; i++){  // put into array
```

Comment:
this comment seems unnecessary, it's not necessary to restate what the code is doing